app/main.py

```diff
@@ -57,20 +58,97 @@
 for r in ROUTERS:
     app.include_router(r)
 
-
 #####################
 ## Middleware
 #####################
 
+gfw_apps = [
+    "https://www.globalforestwatch.org",
+    "https://pro.globalforestwatch.org",
+    "https://forestwatcher.globalforestwatch.org",
+    "https://my.gfw-mapbuilder.org",
+    "cmr.forest-atlas.org",
+    "cmr.atlas-forestier.org",
+    "cmr.forestatlas.org",
+    "cmr.atlasforestier.org",
+    "caf.forest-atlas.org",
+    "caf.atlas-forestier.org",
+    "caf.forestatlas.org",
+    "caf.atlasforestier.org",
+    "rca.atlas-forestier.org",
+    "rca.atlasforestier.org",
+    "cod.forest-atlas.org",
+    "drc.forest-atlas.org",
+    "rdc.atlas-forestier.org",
+    "cod.atlas-forestier.org",
+    "cod.forestatlas.org",
+    "drc.forestatlas.org",
+    "rdc.atlasforestier.org",
+    "cod.atlasforestier.org",
+    "gnq.forest-atlas.org",
+    "gnq.atlas-forestier.org",
+    "gnq.atlas-forestal.org",
+    "gnq.forestatlas.org",
+    "gnq.atlasforestier.org",
+    "cog.forest-atlas.org",
+    "cog.atlas-forestier.org",
+    "cog.forestatlas.org",
+    "cog.atlasforestier.org",
+    "gab.forest-atlas.org",
+    "gab.atlas-forestier.org",
+    "gab.forestatlas.org",
+    "gab.atlasforestier.org",
+    "atlas.mepa.gov.ge",
+    "geo.forest-atlas.org",
+    "geo.forestatlas.org",
+    "lbr.forest-atlas.org",
+    "lbr.forestatlas.org",
+    "mdg.forest-atlas.org",
+    "mdg.atlas-forestier.org",
+    "mdg.forestatlas.org",
+    "mdg.atlasforestier.org",
+    "www.tierrasindigenas.org",
+    "tierrasindigenas.org",
+    "ind.restorationatlas.org",
+    "ind.restoration-atlas.org",
+    "india.restorationatlas.org",
+    "india.restoration-atlas.org",
+    "www.india.restorationatlas.org",
+    "sidhi.restorationatlas.org",
+    "sidhi.restoration-atlas.org",
+    "vp.restorationatlas.org",
+    "vp.restoration-atlas.org",
+    "eth.restorationatlas.org",
+    "eth.restoration-atlas.org",
+    "cmr.amenagement-territoire.org",
+    "cog.amenagement-territoire.org",
+    "cog.reddregistry.org",
+    "cog.registre-redd.org",
+    "www.restauracaovaledoparaiba.org.br",
+    "forum-florestal.doesntexist.org",
+    "monitoramentobahia.dialogoflorestal.org.br",
+]
```

Comment:
Intention here is that only GFW apps (including Mapbuilder) can access Planet tiles using our tile cache and Planet API Key.

I wrote this prior to implementing API keys in Data API. In a future PR will remove this and add API Keys to selected tile caches as well.